tests/test_utils/api_connexion_utils.py

```diff
@@ -49,16 +50,17 @@ def create_role(app, name, permissions=None):
 
 
 def delete_role(app, name):
-    if app.appbuilder.sm.find_role(name):
+    if app.appbuilder.sm.find_role(name) and name not in EXISTING_ROLES:
```

Comment:
With the change on L60 we don't need this too.

```suggestion
    if app.appbuilder.sm.find_role(name):
```